talk/morelanguage.tex

```diff
@@ -177,8 +201,8 @@
     constexpr DimLength km(1, 'k');
     constexpr float km_y = km.get('y');
     constexpr float km_i = km.get('i');
-    std::cout << "1 km = " << km_y << " yards\n"
-              << "     = " << km_i << " inches\n";
+    // static_assert(km_y == 1000, "Expected a km to be 1000 yards"); Fails compilation
+    static_assert(km_y == 1093, "Expected a km to be 1093 yards! ");
```

Comment:
These 2 lines are too long and do not fit in the slide. Try switching to `\small` once more